tests/test-sources/plugins/by-name/hunk/default.nix

```diff
@@ -0,0 +1,100 @@
+{
+  empty = {
+    plugins.hunk.enable = true;
+  };
+
+  with-web-devicons = {
+    plugins.hunk = {
+      enable = true;
+      fileTypeIcons = true;
```

Comment:
CI failure:

> ``error: The option `plugins.hunk.fileTypeIcons' does not exist.``

Did you intend to add an option to `extraOptions`? Or perhaps forgot to update the tests after removing the option?

```suggestion
```